AppInspector.CLI/html/index.html

```diff
@@ -3,17 +3,17 @@
 <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
-    <link rel="stylesheet" href="html/resources/css/bootstrap.min.css" />
-    <link rel="stylesheet" href="html/resources/css/all.css" />
-    <link rel="stylesheet" href="html/resources/css/c3.min.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.5.2/css/bootstrap.min.css" />
```

Comment:
This means you can't view the report offline and it still has external dependencies.  See comment about WebPack for properly minifying assets including dependencies like these.